### ISSUE TITLE
Missing labels for inputs

### ISSUE THREAD

**Issue by aviannamiller** [omeka/plugin-OmekaApiImport#12]
Just about all checkbox and text type inputs are currently missing labels (besides input `omeka_api_import_override_element_set_data`, which is properly labelled).
